docs/advanced/document-store/benchmarking.md

```diff
@@ -0,0 +1,122 @@
+* `ef` (construction): 100
+* `ef` (search): 100
+* `m` (max connections): 16
+
+## Rationale
+Our experiments are designed to be fair and the same across all backends while favouring Document Stores that benefit 
+the DocArray user. Therefore, we based our benchmarks on the following:
+
+* Cover the most important operations: We understand that some backends are better at some operations than others and 
+some offer better quality. Therefore, we try to benchmark on 6 operations (CRUD + Find by vector + Find by condition)
+and report quality measurement (Recall at K).
+* Not just the speed, quality in mind: Since some backends offer better performance and some offer better quality, 
+we make sure to report the quality measurement for the Approximate Nearest Neighbor Search. This will allow users to 
+choose the backend that best suit their cases.
+* Same experiment, same API: DocArray offers the same API across all backends and therefore, we built on top of it the 
+same benchmarking experiment. Furthermore, we made sure to fix the same HNSW parameters for backends that support 
+Approximate Nearest Neighbor Search. All backends are run on official Docker containers, local to the DocArray client 
+which allows having similar network overhead. We also allocate the same resources for those Docker containers and all 
+servers are run in a single node setup.
+* Benefit the user as much as possible: We offer the same conditions and resources to all backends, but our experiment 
+favours the backends that use the resources efficiently. Therefore, some backends might not use the network, or use 
+GRPC instead of HTTP, use batch operations,... And we're okay with that, as long as it benefits the DocArray and Jina 
+user.
+* Open to improvements: We are constantly improving the performance of storage backends from the DocArray side and 
+updating the benchmarks accordingly. If you believe we missed an optimization (perform an operation in batches, benefit 
+from a recent feature in upstream, avoid unnecessary steps,...), feel free to raise a PR or issue, we're open to 
+your contributions!
+
+## Backends covered
+Our benchmarks cover all backends supported by DocArray:
+* In-memory
+* Sqlite
+* Qdrant
+* Annlite
+* Weaviate
+* ElasticSearch
+
+We do not cover the following backends for those reasons:
+* Milvus: currently DocArray does not integrate with Milvus. We're open for contributions to DocArray's repository to 
+support it.
+* Pinecone: Pinecode support is coming soon, and we'll add it to these benchmarks once available.
+* Faiss, Annoy, Scann: We do not benchmark algorithms or ANN libraries. We only benchmark backends that can be used as 
+Document stores. Actually we do not benchmark HNSW itself, but it is used by some backends internally.
+
+
+## Conclusion
+To conclude, you can look at those benchmarks and select the backend that suits better your case.
```

Comment:
This is too weak.

"To concolude, if you are working under ?? Documents, use ??; if you are ...., use ..."